logging-api/src/main/java/com/cisco/oss/foundation/logging/transactions/SchedulerLogger.java

```diff
@@ -47,6 +56,20 @@ public static void success(String schedulerResult) {
 
     }
 
+    public static void successAsync(SchedulerLogger schedulerLogger) {
+
+        FlowContextFactory.deserializeNativeFlowContext(TransactionLogger.getFlowContextAsync(schedulerLogger));
+        schedulerLogger.successInstance();
+
+    }
+
+    public void successAsync() {
+
+        FlowContextFactory.deserializeNativeFlowContext(this.flowContext);
+        this.successInstance();
```

Comment:
you call:
this.successInstance();

doesn't this method also do:
FlowContextFactory.deserializeNativeFlowContext
?